package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@tourstream/fti-group-icon-font",
   "description": "Icon font for projects of FTI Group",
-  "version": "0.12.0",
+  "version": "13.1.0",
```

Comment:
Is this number correct?

Reply:
Of course not. Good point.

Reply:
Hm. It seems that something happened before. (Not in this PR, take a look at master)
The version was changed from 0.14.0 to 13.1.0 before already.
Now we have to maintain it.